doc/dev/ref-manual.md

```diff
@@ -35,15 +35,15 @@ Wherever possible, we reuse existing, high-quality, open-source hardware IP bloc
 For all registers in this section, the functionality is mapped onto the least significant bits of registers and each register is 32 bits wide.
 
 | Base address |    Size | Functionality        |
-|--------------|--------:|----------------------|
-| 0x0010_0000  | 256 KiB | Internal SRAM        |
+|--------------|---------|----------------------|
+| 0x0010_0000  | 128 KiB | Internal SRAM        |
```

Comment:
xbar_main.hjson seems to have the old value?

Reply:
Yes, we can still increase to 256 KiB of SRAM when HyperRAM is disabled: https://github.com/lowRISC/sonata-system/blob/2700cbb6d335188c3d1058f1cdd8319578d65269/rtl/system/sonata_system.sv#L110
However HyperRAM is enabled by default and I want the documentation to show the default configuration.

Reply:
Resolving this is tracked in this issue by the way: https://github.com/lowRISC/sonata-system/issues/228